universal-application-tool-0.0.1/app/services/program/predicate/PredicateValue.java

```diff
@@ -46,16 +45,21 @@ public static PredicateValue of(ImmutableList<String> value) {
             .map(PredicateValue::surroundWithQuotes)
             .collect(toImmutableList())
             .toString(),
-        ScalarType.LIST_OF_STRINGS);
+        OperatorRightHandType.LIST_OF_STRINGS);
+  }
+
+  public static PredicateValue ofListOfLongs(ImmutableList<Long> value) {
```

Comment:
suggestion (non-blocking): how do you feel about naming this `listOfLongs` and the string list one `listOfStrings`?

Reply:
That sounds good to me!